docs/program-setup/bare_metal/baremetal_usage.md

```diff
@@ -0,0 +1,75 @@
+# Using bare metal profile
+
+## Creating a bare metal application
+
+First, clone Mbed OS into the directory where we want to create an application
+```
+mkdir my_mbed_app && cd my_mbed_app
+git clone https://github.com/ARMmbed/mbed-os.git
+```
+
+To enable Mbed OS bare metal, create an `mbed_app.json` with the following contents:
+```
+{
+    "requires": ["bare-metal"]
```

Comment:
Should show small C library use and link to the full explanation

https://os.mbed.com/docs/mbed-os/bare-metal-work/bare-metal/using-small-c-libraries.html
(can see that page on the VPN)

Reply:
That's a good point